app/views/pictures/index.html.slim

```diff
@@ -0,0 +1,32 @@
+- content_for :header do
+  section.hero.is-warning
+    .hero-body
+      .container
+        h1.title
+
+
+.row
+  - @pictures.each do |picture|
+    .card.col-md-4.card-margin
+      .card-img-top.text-center
+        = image_tag(picture.image_url ,style: "width: 18rem;")
```

Comment:
as I said, if we use single quote, then apply it to whole project. Use double quote if we need to pass variable to string